doc/qbk/2.0.quicklook.qbk

```diff
@@ -17,9 +17,7 @@ and interface style of the library. We begin by including the library header fil
 scope.
 
 [c++]
```

Comment:
aren't all .cpp snippets in C++ by default?

Reply:
It depends on the snippet that came before. So unless the snippets are close to each other, I often just write `[c++]` so it doesn't depend on what came before in case we change it and forget to check what comes after.